1-js/08-prototypes/02-function-prototype/article.md

```diff
@@ -64,58 +64,58 @@ Rabbit.prototype = { constructor: Rabbit };
 
 ![](function-prototype-constructor.svg)
 
-We can check it:
+Nós podemos conferir isso:
 
 ```js run
 function Rabbit() {}
-// by default:
+// Por definição:
 // Rabbit.prototype = { constructor: Rabbit }
 
 alert( Rabbit.prototype.constructor == Rabbit ); // true
 ```
 
-Naturally, if we do nothing, the `constructor` property is available to all rabbits through  `[[Prototype]]`:
+Naturalmente, se nós não fizermos nada, a propriedade `constructor` está disponível para todos os coelhos (*rabbits*) através do `[[Prototype]]`:
 
 ```js run
 function Rabbit() {}
-// by default:
+// Por definição:
 // Rabbit.prototype = { constructor: Rabbit }
 
-let rabbit = new Rabbit(); // inherits from {constructor: Rabbit}
+let rabbit = new Rabbit(); // herda de {constructor: Rabbit}
 
-alert(rabbit.constructor == Rabbit); // true (from prototype)
+alert(rabbit.constructor == Rabbit); // true (vindo do protótipo)
 ```
 
 ![](rabbit-prototype-constructor.svg)
 
-We can use `constructor` property to create a new object using the same constructor as the existing one.
+Nós podemos usar a propriedade `constructor` para criar um novo objeto usando o próprio construtor de um objeto que já exista.
```

Comment:
> **Pleonasmo** - [REDUNDANCY_CRIAR](https://community.languagetool.org/rule/show/REDUNDANCY_CRIAR?lang=pt)

- The term "create" already refers to the meaning of "new"
- This way, there is no need for the word "new"
- Change suggestion: `criar um novo` by `criar um`

<img width="380" alt="image" src="https://user-images.githubusercontent.com/35238611/177464847-ffeef3cc-1f8f-4e3e-bef8-ff688d92974b.png">